### PR TITLE
QA-TSWEB-560-Vlad

### PR DESCRIPTION
**Pull request by VladFedash**

### CODE REVIEW

File: ui_tests_playwright/automation/steps/components/button/ButtonSteps.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+import {driver} from '../../../base/driver/Driver';
+
+class ButtonSteps {
+  public async clickButton(type: any, identifier: string, parent?: Locator) {
+    await (await driver.component(type, identifier, parent)).click();
+  }
+}
+
+const buttonSteps = new ButtonSteps();
+
```
Comment (DaniilVostrikov) [inline ui_tests_playwright/automation/steps/components/button/ButtonSteps.ts:11]: remove linebreak

File: ui_tests_playwright/automation/steps/components/container/ContainerSteps.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+  public async checkContainerTitle(identifier: string, expectedTitle: string) {
+    const title = (await driver.component(ContainerById, identifier)).title;
+
+    await expect(title).toHaveText(expectedTitle);
+  }
+}
+
+const containerSteps = new ContainerSteps();
+
```
Comment (DaniilVostrikov) [inline ui_tests_playwright/automation/steps/components/container/ContainerSteps.ts:24]: same

File: ui_tests_playwright/automation/steps/components/job/DescriptionSteps.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+  public async checkDescriptionBlockHeader(identifier: string, expectedTitle: string) {
+    const title = (await driver.component(JobDescriptionBlock, identifier)).title;
+
+    await expect(title).toHaveText(expectedTitle);
+  }
+}
+
+const descriptionSteps = new DescriptionSteps();
+
```
Comment (DaniilVostrikov) [inline ui_tests_playwright/automation/steps/components/job/DescriptionSteps.ts:18]: same

**Review by DaniilVostrikov — CHANGES_REQUESTED** [review]

File: ui_tests_playwright/automation/steps/components/container/ContainerSteps.ts [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+import ContainerById from '../../../components/container/ContainerById';
+
+class ContainerSteps {
+  public async getContainer(type: any, identifier: string) {
+    return await driver.component(type, identifier);
+  }
+
+  public async checkContainerNumber(identifier: string, expectedNumber: string) {
+    const sectionNumber = (await driver.component(ContainerById, identifier)).sectionNumber;
+
```
Comment (DaniilVostrikov) [inline ui_tests_playwright/automation/steps/components/container/ContainerSteps.ts:12]: let's not separate verifications in the very small steps (about 3-4 line)

File: ui_tests_playwright/automation/steps/components/container/ContainerSteps.ts [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+    await expect(sectionNumber).toHaveText(expectedNumber);
+  }
+
+  public async checkContainerTitle(identifier: string, expectedTitle: string) {
+    const title = (await driver.component(ContainerById, identifier)).title;
+
```
Comment (DaniilVostrikov) [inline ui_tests_playwright/automation/steps/components/container/ContainerSteps.ts:18]: same

File: ui_tests_playwright/automation/steps/components/job/DescriptionSteps.ts [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+import JobDescriptionBlock from '../../../components/descriptionBlock/JobDescriptionBlock';
+
+class DescriptionSteps {
+  public async getDescriptionBlock(identifier: string) {
+    return await driver.component(JobDescriptionBlock, identifier);
+  }
+
+  public async checkDescriptionBlockHeader(identifier: string, expectedTitle: string) {
+    const title = (await driver.component(JobDescriptionBlock, identifier)).title;
+
```
Comment (DaniilVostrikov) [inline ui_tests_playwright/automation/steps/components/job/DescriptionSteps.ts:12]: same

File: ui_tests_playwright/automation/test/ui/JobsBlock/JobsBlockPart1.test.ts [hunk 7]
```diff
@@ -53,6 +55,30 @@ test('Check localization on job page @Regression @JobsBlock @TSWEB-560', async (
...
+	);
+
+	const logo = jobPageHeaderContainer.Element.getByTestId(Link.Logo);
+	await logo.waitFor({state: 'visible'});
+
+	await expect(driver.getByTestId(Button.NavigationTab_Jobs)).toHaveText('Jobs');
+	await expect(driver.getByTestId(Button.NavigationTab_AboutUs)).toHaveText('About us');
+	await expect(driver.getByTestId(Button.NavigationTab_Reviews)).toHaveText('Reviews');
+	await expect(driver.getByTestId(Button.NavigationTab_ContactUs)).toHaveText('Contact us');
+
```
Comment (DaniilVostrikov) [inline ui_tests_playwright/automation/test/ui/JobsBlock/JobsBlockPart1.test.ts:74]: mb same?

**Review by DaniilVostrikov — APPROVED** [review]

**Review by anastasiia-liashova — APPROVED** [review]

**Review by YevheniiKel — APPROVED** [review]

**Pull request merged**